README.rst

```diff
@@ -151,6 +151,43 @@ For example, if you wanted to know what code paths query the table
 The performance record herer would include a standard Python traceback attached
 to each SQL query containing "my_table".
 
+
+``capture_operation``, if not ``None``, should be a function that takes one
+argument, the given DB or cache operation, and returns a ``bool`` indiciating if
+the operation should be recorded at all (by default, all operations are
+recorded). Not capturing some operations allows for hiding some code paths to be
+ignored in your tests, such as for ignoring database queries that would be
+replaced by an external service in production.
+
+For example, if you knew that in testing all queries within some function
```

Comment:
I think it would be better to use a simpler first example, for example checking for a particular table name. You could still include the more complex example in an "examples" section at the end.

Reply:
9a95f10 just removes this -- you're right it's rather over complicated for an example.